src/components/Navigation/Navigation.styles.tsx

```diff
@@ -41,16 +41,14 @@ export const StyledFontAwesomeIcon = styled(FontAwesomeIcon)`
     color: ${themes.color.secondary};
 `;
 
-export const SingleNavItem = styled(Link)<SingleNavItemProps>`
-    display: ${props => (props.isShow ? 'block' : 'none')};
+export const SingleNavItem = styled.div<SingleNavItemProps>`
     padding: 0 5px;
+    display: ${props => (props.isShow ? 'bloc' : 'none')};
     background-color: ${themes.color.primaryLight};
-    color: inherit;
-    text-decoration: none;
-    transition: background-color 0.3s;
     &:hover {
         background-color: ${themes.color.secondary};
```

Comment:
Do you will use multiple themes objects in future? Cause name `themes` is liitle bit confusing. Maybe you should name it simple `theme`?